src/scenes/home/home.js

```diff
@@ -230,6 +242,11 @@ class Home extends Component {
               )}
             />
             <AuthenticatedRoute
+              exact path="/scholarships/:id/apply"
```

Comment:
How will `:id` work here?

Reply:
It's used to fetch scholarship information among other things